notebooks/Tutorial10_Geolocation_privacy_assessment.py

```diff
@@ -87,7 +80,7 @@
 plt.show
 
 # +
-selected_id = 1
+selected_id = "id1"
```

Comment:
Also I disagree with this. We should not have to change the data file. We need to go and fix the ID handling, for sure we'll get numeric IDs in practice.